### DISCUSSION TITLE
Crash for iOS when run Ubuntu in all gpu model. Thought other tool found UTM memory leak when the gpu model. Please fix it.

### DISCUSSION THREAD

**Discussion by liaoinvest** [utmapp/UTM#6226] [UTM Help (iOS)]
iOS 16.3
TrollStore
UTM 4.4.5
iPad Pro 2022 1t
 

Video:

https://github.com/utmapp/UTM/assets/89998407/cdd5b928-359b-4c70-8c79-0b8bd8a3e871

**Comment by osy** [comment]
This isn't a memory "leak" but just that UTM/ANGLE is very unoptimized and uses a lot more memory than required. You just need to assign less guest RAM and turn down the resolution.
